PureBasicIDE/Common.pb

```diff
@@ -2862,6 +2862,78 @@ CompilerIf #PB_Compiler_Debugger
   Global InDebuggerCallback = #False
 CompilerEndIf
 
+; Ascii only MemoryStringCompare to have much faster compare as we don't need unicode support for token parsing (https://www.purebasic.fr/english/viewtopic.php?t=78887)
+;
+Global  Dim NoCaseAsciiLookup(255)
```

Comment:
If that array only contain ASCII chars, shouldn't it be declared as `Global  Dim NoCaseAsciiLookup.c(255)`?